lib/function.g

```diff
@@ -191,8 +191,9 @@ DeclareOperationKernel( "SetNameFunction", [IS_OBJECT, IS_STRING], SET_NAME_FUNC
 ##  <Description>
 ##  returns the number of arguments the function <A>func</A> accepts.
 ##  -1 is returned for all operations.
-##  For functions that use <C>arg</C> to take a variable number of arguments,
-##  the number returned is - the total number of parameters including the <C>arg</C>.
+##  For functions that use <C>...</C> or <C>arg</C> to take a variable number
+##  of arguments,
+##  the number returned is - the total number of parameters.
```

Comment:
Why is this on three lines, instead of three?

BTW, minor nitpick: I find the text "... is - the total number" quite confusing (I realize that it was there before, but I already found it confusing then). How about changing it to "... is -1 times the total number", while we are at it?